packages/example-api-gateway/src/readBlockchain.ts

```diff
@@ -0,0 +1,23 @@
+import { createWatcher } from '@aloxide/demux';
+
+import config from './config';
+
+createWatcher({
+  accountName: 'helloworld12',
+  modelBuilderConfig: {
+    aloxideConfigPath: config.aloxideConfigPath,
+    logger: config.logger,
+  },
+  sequelize: config.sequelize,
+  nodeActionReaderOptions: {
+    nodeosEndpoint: process.env.app_nodeosEndpoint,
+    onlyIrreversible: false,
+    startAtBlock: 9130000,

```

Comment:
the start block should be loaded from constants file or env file?

Reply:
Fixed!